config/core/resources/cloudauditlogssource.yaml

```diff
@@ -36,158 +36,297 @@ spec:
     kind: CloudAuditLogsSource
     plural: cloudauditlogssources
   scope: Namespaced
-  subresources:
-    status: {}
   preserveUnknownFields: false
   conversion:
     strategy: Webhook
-    webhookClientConfig:
-      service:
-        name: webhook
-        namespace: cloud-run-events
-  additionalPrinterColumns:
+    webhook:
+      # conversionReviewVersions indicates what ConversionReview versions are understood/preferred by the webhook.
+      # The first version in the list understood by the API server is sent to the webhook.
+      # The webhook must respond with a ConversionReview object in the same version it received.
+      conversionReviewVersions: ["v1beta1", "v1", "v1alpha1"]
```

Comment:
@Harwayne @nachocano I'm not quite sure about this... We should make "v1" the first one after v1 is the storage version. But currently "v1beta1" is the storage version, so it'd be safer that we keep "v1beta1" as the first one in the list?

Reply:
Based on the doc https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/ and the examples in knative eventing, https://github.com/knative/eventing/blob/master/config/core/resources/sinkbindings.yaml
I think it should be `conversionReviewVersions: ["v1beta1", "v1"]`.
I don't think it's related to the version of sources, it's related to the versions of  `apiextensions.k8s.io`.
@Harwayne  @nachocano  Please correct me if I'm wrong.

Reply:
similar issues for other resources.

Reply:
I see, I fixed it in the current commit.